challenges/objects-arrays.js

```diff
@@ -114,12 +267,15 @@ The zoos need to know their total animal population across the United States.  F
 
 */
 const populationTotal = 0;
```

Comment:
again remember to comment out this line so your code works
const populationTotal = 0;